qa/suites/rbd/mirror-thrasher/base

```diff
@@ -0,0 +1 @@
+../mirror/base
```

Comment:
@trociny (NIT) Please consider to make title of PR<50 characters
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits